### DISCUSSION TITLE
Still no migrations files

### DISCUSSION THREAD

**Discussion by VicToMeyeZR** [thedevdojo/voyager#5516] [General]
Why does voyager still not create migrations files for the daatbase?

**Comment by VicToMeyeZR** [comment]
I mean.  If you're gonna call yourself a laravel applicaton, you should FOLLOW their conventions..

**Comment by emptynick** [comment]
99.9% of the time this question can be answered with: because **you** didn't implement it yet.

https://twitter.com/taylorotwell/status/1256031217943818242?t=l_G496VXdO21FNt-BuRG1w&s=19

Seriously, this is an open source project and no one of is being paid for working on Voyager.

**Reply by howdu** [reply]
People are always too quick to complain at the littlest things and even when it's free. You guys do awesome job thank you!

**Reply by VicToMeyeZR** [reply]
This is a COMPLETELY VALID feedback.  If you're going to call yourself a laravel app, you should follow laravel basic conventions...   It's that simple

**Reply by emptynick** [reply]
In my eyes this is just an allegation against the team and probably the whole open-source community.
We don't call ourselfs a Laravel app nonetheless.
Also, only because migrations exist in Laravel there is absolutely no reason why we have to implement them.
We don't implement other things existing in Laravel as well and there is no problem with that.

**Reply by VicToMeyeZR** [reply]
You don't call yourself a Laravel app?  "The Missing Laravel Admin"..  
I'm not slamming open-source community.

**Reply by ousid** [reply]
If you don't like it, don't use it.
Simple as that :)

**Comment by VicToMeyeZR** [comment]
If I was a good enough developer to do that, I wouldn't need Voyager would I?
